webtau-open-api/src/main/java/com/twosigma/webtau/openapi/OpenApiSpecConfig.java

```diff
@@ -34,6 +37,14 @@
 
     private static String fullPath;
 
+    public static ConfigValue getSpecUrl() {
```

Comment:
we probably should rename this to `getSpecUrlConfigValue` to be consistent with the other values exposed this way.